packages/twitch-chat-client/src/ChatUser.ts

```diff
@@ -47,6 +47,19 @@ export default class ChatUser {
 		return new Map(badgesStr.split(',').map(badge => badge.split('/', 2) as [string, string]));
 	}
 
+	/**
+	 * The badge info of the user. Returned as a map that maps the badge category to the detail.
+	 */
+	get badgeInfo() {
+		const badgeInfoStr = this._userData.get('badge-info');
+
+		if (!badgeInfoStr) {
+			return new Map<string, string>();
+		}
+
+		return new Map(badgeInfoStr.split(',').map(badge => badge.split('/', 2) as [string, string]));
```

Comment:
I would appreciate if you put this into a private helper method instead of copying it 😄